airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json

```diff
@@ -8,12 +8,13 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "BigQuery Destination Spec",
     "type": "object",
-    "required": ["project_id", "dataset_id"],
+    "required": ["project_id", "dataset_id", "big_query_client_buffer_size_mb", "dataset_location",
```

Comment:
@sherifnada I reversed these changes. Then how do I notice these fields, since, they have a default value, but, add them to the list of required we can not? I can add a star to the field name, but I don't think that's a very good option.

Reply:
if the field is optional, then the connector should be able to handle a JSON which doesn't contain that field and instead use its own default values. Do you think that's doable in this case? IMO it's fine if we don't add a star or anything, and just leave it like a completely optional field

Reply:
Got you. I corrected this, added an optional note to the title of these fields, and in the description of these fields, specified the default value.